### PR TITLE
Added list of maintainers 

### PR DESCRIPTION
**Pull request by drashti4**
- Highlighted contribution area
- Added maintainer list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,18 +124,22 @@ To be done
...
-[CHAOSS Software](https://lists.linuxfoundation.org/mailman/listinfo/chaoss-software) mailing lists explaining
-a bit your interest.
+The collaboration guide is still in progress, but a first good step would be to introduce yourself at the [CHAOSS Metrics](https://lists.linuxfoundation.org/mailman/listinfo/oss-health-metrics) or [CHAOSS Software](https://lists.linuxfoundation.org/mailman/listinfo/chaoss-software) mailing lists explaining a bit your interest.
 
 Most of the work in the metrics side will be done through pull requests to the markdown files containing discussions
 about the metrics used or to be used. This is the same method as when updating the software side of the project.
 
+## Maintainers
+
+- [Georg J.P. Link](https://github.com/GeorgLink)
```
Comment (GeorgLink) [inline README.md]: Line 137: I would prefer to NOT be a maintainer. I appreciate the acknowledgement.

**Comment by GeorgLink** [comment]
fixes #6

**Comment by GeorgLink** [comment]
fixes #6

**Comment by drashti4** [comment]
Done :+1:

File: README.md [hunk 2]
```diff
@@ -124,18 +124,20 @@ To be done
...
+The collaboration guide is still in progress, but a first good step would be to introduce yourself at the [CHAOSS Metrics](https://lists.linuxfoundation.org/mailman/listinfo/oss-health-metrics) or [CHAOSS Software](https://lists.linuxfoundation.org/mailman/listinfo/chaoss-software) mailing lists explaining a bit your interest.
 
 Most of the work in the metrics side will be done through pull requests to the markdown files containing discussions
 about the metrics used or to be used. This is the same method as when updating the software side of the project.
 
+## Maintainers
+
+- [Daniel Izquierdo Cortazar](https://github.com/dicortazar)
+
+If you'd like to our squad, just go through the [issue list](https://github.com/chaoss/wg-diversity-inclusion/issues) and fix some. :tada:
```
Comment (GeorgLink) [inline README.md]: The first part of the sentence on line 139 reads funny.

How about:
If you would like to **be on** our squad, ... ?

**Comment by GeorgLink** [comment]
Thanks @drashti4

**Comment by drashti4** [comment]
Oh correct! I always try to create more friendly README rather than professional for opensource project.

**Comment by GeorgLink** [comment]
Thanks! I am perfectly happy with 'more friendly' :-)

**Comment by drashti4** [comment]
Do you think it's ready for merge?

**Comment by GeorgLink** [comment]
​I give it a thumbs up but defer to repository maintainers.

**Comment by emmairwin** [comment]
Is there a list of contributors somewhere?  We should have that

**Comment by GeorgLink** [comment]
​No, we do not. I like the idea!

**Comment by dicortazar** [comment]
Thanks a lot for this PR! This looks good to me as well.

I'll open a new ticket to add the list of contributors. And in any case those are listed at [1], hope this helps :).

And then I'll merge this one.

[1] https://chaoss.biterg.io/goto/c2742cf386c1e2fed0fc2d862aeeffce

**Pull request closed without merging**